lib/recurly/token.js

```diff
@@ -145,5 +145,15 @@ function validate (input) {
     errors.push('last_name');
   }
 
+  if (input.cvv && !this.validate.cvv(input.cvv)) {
+    errors.push('cvv');
+  }
```

Comment:
Ah okay, I see now how this is first checking for input presence. Okay, that looks great!